quick_start.sh

```diff
@@ -0,0 +1,159 @@
+#!/bin/bash
+set -euo pipefail
+
```

Comment:
We could fail right away if OS==Windows? Might not be worth the difficulty though as a) we don't expect windows users, and b) I don't know how WSL reports e.g. the `$OSTYPE` variable...

Just thinking out loud really

Reply:
```bash
if uname -s | grep -qE "CYGWIN|MINGW|MSYS"; then
    echo "Quick start does not support Windows"
    exit 1
fi
```

Reply:
Added a few windows checks, but I have not easy way to test them. So, flying blind a little.